packages/mcfunction/src/colorizer/macro.ts

```diff
@@ -0,0 +1,39 @@
+import * as core from '@spyglassmc/core'
+import type { MacroNode } from '../node/macro'
+
+export const macro: core.Colorizer<MacroNode> = (node, ctx) => {
```

Comment:
(from discord)

> Got to admit I don't understand the Colorizer unit tests at all so I'm going to avoid that for now

admittedly I haven't touched the colorizer before nor its unit tests, and it looks simple enough so its probably fine to skip unit tests for it 👍